experimental/packages/exporter-logs-otlp-http/package.json

```diff
@@ -101,13 +101,14 @@
     "webpack-merge": "5.9.0"
   },
   "peerDependencies": {
-    "@opentelemetry/api-logs": ">=0.38.0"
+    "@opentelemetry/api": "^1.0.0"
   },
   "dependencies": {
     "@opentelemetry/core": "1.15.0",
     "@opentelemetry/otlp-exporter-base": "0.41.0",
     "@opentelemetry/otlp-transformer": "0.41.0",
     "@opentelemetry/sdk-logs": "0.41.0",
+    "@opentelemetry/api-logs": "0.41.0",
```

Comment:
I think this should still be a peer dependency?

Reply:
That would also be an option. :thinking: 

Though I think we'd need to peer depend on a strict `0.41.0` anyway (or `^0.41.0`, but bugfix releases will bump all packages anyway). When working on metrics, we've done it like proposed here and used the direct dependency as we needed to have that exact package (the API might still break) - making it a peer dependency would have added little value. :thinking:

I can still change it if you prefer, thought then we'd have to do the same with the grpc and proto exporters :slightly_smiling_face: